src/lib/Microsoft.Health.Fhir.Ingest/Service/MeasurementFhirImportService.cs

```diff
@@ -24,7 +24,7 @@ public class MeasurementFhirImportService : ParallelTaskWorker<MeasurementFhirIm
         private readonly FhirImportService _fhirImportService;
 
         public MeasurementFhirImportService(FhirImportService fhirImportService, MeasurementFhirImportOptions options)
-            : base(options, options.ParallelTaskOptions.MaxConcurrency)
+            : base(options, PassThroughNonNull(options).ParallelTaskOptions.MaxConcurrency)
         {

```

Comment:
Can probably replace with if null conditional ? and default to 1 if null.

Reply:
Also, Ensure.IsNotNull should return the value so you can just inline that with out needing your own function.